docs/edgeql/operators/json.rst

```diff
@@ -65,3 +80,57 @@ The contents of JSON *objects* and *arrays* can also be accessed via ``[]``:
     ... })
     ... FILTER j['name'] = <json>'std::bool';
     {{name: 'std::bool', timestamp: '2018-10-18'}}
+
+The element access operator ``[]`` will raise an exception if the
+specified index is not valid or does not exist for the base JSON
+value. It is possible to handle invalid indexes without raising an
+exception with the help of :eql:func:`json_get`.
+
+
+JSON Array Slice
+================
+
+JSON arrays can be sliced in the same way as regular arrays, producing
+in a new JSON array:
+
+.. code-block:: edgeql-repl
+
+    db> SELECT to_json('[1, 2, 3]')[0:2];
+    {[1, 2]}
+
+    db> SELECT to_json('[1, 2, 3]')[2:];
+    {[3]}
+
+    db> SELECT to_json('[1, 2, 3]')[:1];
+    {[1]}
+
+    db> SELECT to_json('[1, 2, 3]')[:-2];
+    {[1]}
+
+
+Casting Objects into JSON
+=========================
+
+There's one more way of getting a JSON value and that is by casting
```

Comment:
Simplify to ":eql:type:`Object` can be cast into JSON"?  This whole sentence is hard to read/understand; too many words.

Reply:
Rephrased.